.github/workflows/auto_update.yml

```diff
@@ -2,7 +2,7 @@ name: Auto Update
 
 on:
   schedule:
-    - cron: '10 10 15 * *'
+    - cron: '0 0 * * 0' # Run every Sunday at midnight
```

Comment:
I think we don't want to run `auto_update` on a weekly basis, once a month is enough...

Reply:
sure, let me change it